pkg/extension/controller/builder.go

```diff
@@ -121,6 +129,16 @@ func (b *Builder) Build() (*ExtensionController, error) {
 		watchSources = append(watchSources, source)
 	}
 
+	for _, obj := range b.ownTypes {
```

Comment:
It was somewhat tricky to verify that this did only _Owned_ resources, but I changed the subscribe from true to false for verification and could see that only owned routes triggered the OIDCPolicy reconcile, good stuff!